elements/campaign-display/fields/campaign-request-field.js

```diff
@@ -19,7 +19,7 @@ const CampaignRequestField = {
 
     fetchCampaignSuccess(state, response, store) {
       state.requestInFlight = false;
-      store.actions.handleFetchComplete(response);
+      setImmediate(() => store.actions.handleFetchComplete(response));
```

Comment:
How do we feel about a constraint that an action MAY NOT call out to a second action on the same field.

If we can lock down nested actions to only being on other fields, we can avoid any kind of setImmediate/queueing.

In this case, I imagine calling out a second action on the same field would throw an error.